Source/Layout/ASStackLayoutSpec.mm

```diff
@@ -160,7 +160,7 @@ - (ASLayout *)calculateLayoutThatFits:(ASSizeRange)constrainedSize
     self.style.descender = stackChildren.back().style.descender;
   }
   
-  auto sublayouts = [[NSMutableArray<ASLayout *> alloc] init];
+  const auto sublayouts = [[NSMutableArray<ASLayout *> alloc] init];
```

Comment:
Should this be let and same below?

Reply:
I deliberately didn't move the stack layout spec over to the new syntax yet as most of the stack layout specs are c++ from the beginning on and used auto in there.